pyproject.toml

```diff
@@ -72,6 +72,7 @@ tldextract = ">=5.1.0"
 typer = ">=0.12.0"
 typing-extensions = ">=4.1.0"
 yarl = ">=1.18.0"
+rich = "^13.9.4"
```

Comment:
Could you please use ">=" version specifier? And please put it in alphabetical order :slightly_smiling_face:.